ethToElrond/v2/elrondToEth/steps/step07SignProposedSetStatus.go

```diff
@@ -0,0 +1,62 @@
+package steps
+
+import (
+	"context"
+
+	"github.com/ElrondNetwork/elrond-eth-bridge/core"
+	v2 "github.com/ElrondNetwork/elrond-eth-bridge/ethToElrond/v2"
+	"github.com/ElrondNetwork/elrond-eth-bridge/ethToElrond/v2/elrondToEth"
+)
+
+type signProposedSetStatusStep struct {
+	bridge elrondToEth.ElrondToEthBridge
+}
+
+func (step *signProposedSetStatusStep) Execute(ctx context.Context) (core.StepIdentifier, error) {
+	batch := step.bridge.GetStoredBatch()
+	if batch == nil {
+		step.bridge.GetLogger().Debug("no batch found")
+		return elrondToEth.GettingPendingBatchFromElrond, nil
+	}
+
+	actionID, err := step.bridge.GetAndStoreActionIDForSetStatusFromElrond(ctx)
```

Comment:
there will be an intermediate step here, forgot about it :(
The step in which we get the statuses of the transfers executed on Ethereum and apply them on the current stored batch.